automations/armcontroller.py

```diff
@@ -0,0 +1,19 @@
+from components.bunnygrabber import BunnyGrabber
+from components.armextention import ArmExtention
+
+
+class ArmController:
```

Comment:
What is this for?

Reply:
That shouldn’t be there, it shouldn’t have gone in the commit.